BetaCameras/BaslerACA/BaslerACA.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Metrilus GmbH
+// MetriCam 2 is licensed under the MIT license. See License.txt for full license text.
+
+using Metrilus.Util;
+using System;
+using System.Linq;
+using System.Drawing;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using PylonC.NETSupportLibrary;
+using PylonC.NET;
+
+namespace MetriCam2.Cameras
+{
+    public class BaslerACA : Camera
+    {
+        private const string DeviceClass = "acA1300";
```

Comment:
DeviceClass is just a string used to filter the list of cameras that the Pylon library spits out. We would support all cameras when we don't use it at all. But since I'm unsure what the other cameras supported by Pylon are capable of, I check the camera name right now.

Reply:
Let me rephrase: "if it were configurable we would directly support all cameras compatible with acA1300"

Reply:
So make it a public property with the default value of "acA1300" so a developer can change it when he knows what he is doing, but GUI applications would still work only with tested cameras.
Probably makes sense to change this to a list of supported devices.